res/database/Default/lang/de.xml

```diff
@@ -155,10 +155,47 @@
 		<programmerole guid="6e48da2f-d468-4542-9510-c3412e8841f7" first_name="" last_name="Wirsch" title="Kommissar" />
 		<programmerole guid="3b128a1f-c2e5-4f76-aa97-0d23d44568d2" first_name="Darlene" last_name="" title="Assistentin" />
 	</programmeroles>
-
-<!--Define global variables here
 	<globalvariables>
-		<variableName>value de</variableName>
+
+		<!--
+			Marken, Sportvereine, Musiktitel und ähnliche Ressourcen, die in Nachrichten, Ads, Sendungen usw. erscheinen können, werden von hier referenziert.
+			Die englische Datei ist die Hauptreferenz, die alle Einträge enthält.
+			Ab hier sind Übersetzungen in andere Sprachen ad-lib. Die Einträge müssen nicht mehr in jeder Sprache gespiegelt werden.
+		-->
+
+		<brand_windows>Windoof</brand_windows>
+		<brand_xbox>Ykasten</brand_xbox>
+		<brand_encarta>Enzyklopädie</brand_encarta>
+		<brand_commodore>Cormorano</brand_commodore>
+		<brand_deutschetelekom>Deutsche Telekrampf</brand_deutschetelekom>
+		<brand_telekom>Telekrampf</brand_telekom>
+		<brand_beetle>Käfli</brand_beetle>
+		<brand_reebok>Rehböck</brand_reebok>
+		<brand_mars>Morss</brand_mars>
+		<brand_toblerone>Tubenkrone</brand_toblerone>
+		<brand_puschkin>Jutschin</brand_puschkin>
+
+		<generalorg_communistpartyofthesovietunion>Kommunistische Partei der Vereinigten Sowjetstaaten</generalorg_communistpartyofthesovietunion>
+		<generalorg_ofthecommunistpartyofthesovietunion>der Kommunistischen Partei der Vereinigten Sowjetstaaten</generalorg_ofthecommunistpartyofthesovietunion>
+		<generalorg_cpsu>KPdvSS</generalorg_cpsu>
+		<generalorg_socialdemocraticpartyofgermany>Sozialdemokratische Populistenpartei</generalorg_socialdemocraticpartyofgermany>
+		<generalorg_socialistunitypartyofgermany>Sozialistisch-Deutsche Einheitspartei</generalorg_socialistunitypartyofgermany>
+		<generalorg_ofthesocialistunitypartyofgermany>der Sozialistisch-Deutsche Einheitspartei</generalorg_ofthesocialistunitypartyofgermany>
+
+		<sports_formula1>Formel A</sports_formula1>
+		<sports_formula2>Formel B</sports_formula2>
+		<sports_formula3>Formel C</sports_formula3>
+		<sports_formula4>Formel D</sports_formula4>
+
+		<mediageneralorg_worldwrestlingfederation>Westliche Wrestling-Freunde</mediageneralorg_worldwrestlingfederation>
+		<mediageneralorg_theworldwrestlingfederation>die Westlichen Wrestling-Freunde</mediageneralorg_theworldwrestlingfederation>
+
+		<mediatitle_lowestofthelow>Voll am Boden</mediatitle_lowestofthelow>
```

Comment:
I'm doing it the other way round now, referencing in programme entries from the global variables. That's because program titles often have addendums (especially series with "Season 1" etc currently) that are not wanted in other contexts. Maybe one day we have a "franchise" structure and can reference from there.

Reply:
OK